tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend_test_ci-pipeline.yml

```diff
@@ -0,0 +1,32 @@
+trigger: none
+
+jobs:
+- template: templates/linux-ci.yml
+  parameters:
+    AgentPool : 'ORT-Training-Frontend-Linux-GPU'
```

Comment:
this agent pool is not in https://dev.azure.com/onnxruntime/_settings/agentpools, and not a public one
we can configure (override) it directly from the build definition and not specify it here